cafebabel/templates/articles/detail.html

```diff
@@ -3,8 +3,12 @@
 
 {% block meta_title %}{{ article.title }}{% endblock %}
 {% block meta_description %}{{ article.summary }}{% endblock %}
+{% block meta_image_absolute_url %}{{ absolute(article.image_url) }}{% endblock %}
+{% block meta_author %}{{ article.author }}{% endblock %}
```

Comment:
By default, the `article.author` renders the email of the author if he didn't mention a name. That could be spammy! 💣

Reply:
Yep, maybe we shouldn't allow a profile creation without the full name after all.. what do you think ?

Reply:
Or not use the email as a default name but something engaging? 🤔

Reply:
What to you mean engaging ?

Reply:
If you call your user `Pinky Lion` or `Crazy Omelette` or whatever it encourages the person to change it right away 😉